tests/kafkatest/tests/produce_consume_validate.py

```diff
@@ -51,7 +51,7 @@ def stop_producer_and_consumer(self):
 
         # Check that producer is still successfully producing
         currently_acked = self.producer.num_acked
-        wait_until(lambda: self.producer.num_acked > currently_acked + 5, timeout_sec=10,
+        wait_until(lambda: self.producer.num_acked > currently_acked + 5, timeout_sec=360,
```

Comment:
This seems like a huge bump and looks suspiciously like a metadata timeout.

Is there an associated JIRA for the underlying problem? Patches like this should probably either a) have an explanation for why the timeout *must* be very high (if it isn't due to a bug) or b) have a comment with the associated JIRA (and note in the JIRA to reduce the timeout when the bug is fixed) so we don't just accumulate huge timeouts with no explanation for how they can be reduced.